examples/storage/src/main.rs

```diff
@@ -13,91 +13,100 @@ use ariel_os::debug::{
 use ariel_os::storage;
 use serde::{Deserialize, Serialize};
 
-// Some object. For storing it, derive the serde Serialize / Deserialize traits.
+// Example object.
+// The serde Serialize / Deserialize traits are required for storage
 #[derive(Serialize, Deserialize, Debug, defmt::Format)]
 struct MyConfig {
-    val_uno: heapless::String<64>,
-    val_dos: u64,
+    val_one: heapless::String<64>,
+    val_two: u64,
 }
 
 #[ariel_os::task(autostart)]
 async fn main() {
-    info!("Hello from storage test!");
+    info!("Start storage example");
 
     // Storing a primitive type (e.g., u32)
     let value: Option<u32> = storage::get("counter").await.unwrap();
     let value = if let Some(value) = value {
-        info!("got counter value {}", value);
+        info!("got counter value {} from storage", value);
         value
     } else {
-        info!("no counter value. first time running this test? try rebooting.");
+        info!("no counter value in storage. Is this the first time running this example?");
         0
     };
 
     if value > 10 {
-        info!("counter value > 10, aborting test to safe flash cycles.");
+        info!("counter value > 10, aborting test to save flash cycles");
         exit(ExitCode::SUCCESS);
     }
+    info!("");
 
     storage::insert("counter", value + 1).await.unwrap();
 
+    // By getting the Storage mutex directly, changing e.g., a counter,
+    // can be done atomically w.r.t. concurrent access from the same firmware:
     {
-        // By getting the Storage mutex directly, changing e.g., a counter,
-        // can be done atomically w.r.t. concurrent access from the same firmware:
         let mut s = storage::lock().await;
-        let value: Option<u32> = s.get("counter").await.unwrap();
+        let value: Option<u32> = s.get("atomic_counter").await.unwrap();
```

Comment:
The comment explains what is meant with atomicity; yet maybe calling it atomic counter is maybe not ideal, as there are other aspects to atomicity.

Or: if the counter is atomic in the sense that after insert completes, we are guaranteed not to read the old value any more at the next reboot, maybe we could comment on that and then it's atomic for all senses I could come up with.